src/vulkan/device.h

```diff
@@ -30,7 +30,7 @@ class Device {
   explicit Device(VkDevice device);
   ~Device();
 
-  Result Initialize();
+  Result Initialize(const VkPhysicalDeviceFeatures& features);
```

Comment:
are these the required_features or the available features? (and below) can you rename to make clear?

Reply:
updated